dashboard/app/config.go

```diff
@@ -128,6 +128,12 @@ type Config struct {
 	ReproExportPath string
 }
 
+type ACLItem struct {
+	Email  string
+	Domain string
```

Comment:
Could `Domain` be confused with `trustedAuthDomain`?
Please add comments, even better with some examples.

Reply:
Thanks. Let's document it.